sources/packages/backend/libs/sims-db/src/entities/notification.model.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  OneToOne,
+  PrimaryColumn,
+  PrimaryGeneratedColumn,
+} from "typeorm";
+import { ColumnNames, TableNames } from "../constant";
+import { Message } from "./message.model";
+import { RecordDataModel } from "./record.model";
+import { User } from "./user.model";
+
+@Entity({
+  name: TableNames.Notifications,
+})
+export class Notification extends RecordDataModel {
+  /**
+   *  Primary key column.
+   */
+  @PrimaryColumn({
+    name: "id",
+    nullable: false,
+  })
+  id: number;
```

Comment:
@PrimaryGeneratedColumn() ?

Reply:
I have updated it, I made a mistake here.